src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -385,28 +384,9 @@ private void InjectAdditionalFiles()
         {
             foreach (var project in this.projectSystemHelper.GetSolutionManagedProjects())
             {
-                string projectRoot = PathHelper.ForceDirectoryEnding(Path.GetDirectoryName(project.FullName));
-
-                MSBuild.Project msBuildProject = this.projectSystemHelper.GetEquivalentMSBuildProject(project);
-
-                Debug.Assert(msBuildProject != null, "Couldn't find equivalent MSBuild project object to VS project object");
-
-                if (msBuildProject != null)
+                foreach (var additionalFilePath in this.AdditionalFilePaths)
```

Comment:
Minor optimisation, unrelated to the bug: there's no point in walking the projects in the solution if there are no additional files. The calling method _InjectAdditionalFiles_ could early out if there are no files to add.